elasticdl/python/tests/mock_kv_store.py

```diff
@@ -23,11 +27,16 @@ def lookup(self, keys=[], *args, **kwargs):
                 unknown_key.append(i)
         return values, unknown_key
 
+    # keep same with EmbeddingService.update_embedding
```

Comment:
What does this comment mean? You can make this a docstring instead of a comment

Reply:
Done.